src/libs/ReportActionsUtils.ts

```diff
@@ -1199,6 +1199,13 @@ function isLinkedTransactionHeld(reportActionID: string, reportID: string): bool
     return TransactionUtils.isOnHoldByTransactionID(getLinkedTransactionID(reportActionID, reportID) ?? '');
 }
 
+/**
+ * Check if the current user is the requestor of the action
+ */
+function isRequestor(reportAction: OnyxEntry<ReportAction>): boolean {
```

Comment:
Sorry I kinda missed this in the proposal and original review, but I was thinking about reports when thinking about `isRequestor`, but for report actions, this name isn't great. 

How about we call this something like `ReportActionUtils.actionWasTakenByCurrentUser(reportAction)`

Reply:
Cool, let me know when this is ready for re-review. Thanks